app/src/androidTest/java/ch/epfl/sweng/erpa/services/DummyUserProfileServiceTest.java

```diff
@@ -0,0 +1,54 @@
+package ch.epfl.sweng.erpa.services;
+
+import android.support.test.InstrumentationRegistry;
+import android.support.test.runner.AndroidJUnit4;
+
+import com.annimon.stream.Optional;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import ch.epfl.sweng.erpa.model.UserProfile;
+import ch.epfl.sweng.erpa.services.dummy.database.DummyUserService;
+import ch.epfl.sweng.erpa.util.TestUtils;
+
+import static ch.epfl.sweng.erpa.util.TestUtils.getUserProfile;
+import static ch.epfl.sweng.erpa.util.TestUtils.unnecessaryCodeClimateMethod;
+import static junit.framework.TestCase.assertEquals;
+import static junit.framework.TestCase.assertTrue;
+
+@RunWith(AndroidJUnit4.class)
+public class DummyUserProfileServiceTest {
+
+    DummyUserService ups;
+
+    @Before
+    public void initDB() {
+        ups = new DummyUserService(InstrumentationRegistry.getTargetContext());
```

Comment:
No, you should ask the di injection framework for an instance: you want to have as close as possible conditions and don't forget to reset each time